src/main/java/com/sprint/mission/discodeit/DiscodeitApplication.java

```diff
@@ -0,0 +1,99 @@
+package com.sprint.mission.discodeit;
+
+import com.sprint.mission.discodeit.repository.ChannelRepository;
+import com.sprint.mission.discodeit.repository.MessageRepository;
+import com.sprint.mission.discodeit.repository.file.FileChannelRepository;
+import com.sprint.mission.discodeit.repository.file.FileMessageRepository;
+import com.sprint.mission.discodeit.service.AuthService;
+import com.sprint.mission.discodeit.service.ChannelMessageService;
+import com.sprint.mission.discodeit.service.ChannelService;
+import com.sprint.mission.discodeit.service.MessageService;
+import com.sprint.mission.discodeit.service.UserService;
+import com.sprint.mission.discodeit.test.AuthTest;
+import com.sprint.mission.discodeit.test.ChannelTest;
+import com.sprint.mission.discodeit.test.MessageTest;
+import com.sprint.mission.discodeit.test.UserTest;
+import org.springframework.boot.SpringApplication;
+import org.springframework.boot.autoconfigure.SpringBootApplication;
+import org.springframework.context.ConfigurableApplicationContext;
+
+import java.io.IOException;
+import java.util.UUID;
+
+@SpringBootApplication
+public class DiscodeitApplication {
+
+    public static void main(String[] args) throws IOException {
+        ConfigurableApplicationContext context = SpringApplication.run(DiscodeitApplication.class, args);
+
+        /**
+         * User Test
+         */
+
+        // 필요한 인스턴스 생성
+        UserService userService = (UserService) context.getBean("basicUserService");
```

Comment:
구체 클래스명을 적어주기 보다는 인터페이스에 의존하는게 좋을 것 같아요.  
만약 basicUserService 클래스명이 변경된다면 테스트가 정상동작하지 않을 거예요.

```suggestion
        UserService userService = context.getBean(UserService.class);
```